src/bidiMapper/domains/context/BrowsingContextProcessor.ts

```diff
@@ -415,7 +393,19 @@ export class BrowsingContextProcessor {
           break;
         }
 
-        this.#handleWorkerTarget(cdpTarget, browsingContext.id);
+        const cdpTarget = this.#createCdpTarget(
+          targetCdpClient,
+          targetInfo,
+          sessionId
+        );
+        this.#handleWorkerTarget(
+          cdpTarget,
+          this.#realmStorage.getRealm({
+            browsingContextId: browsingContext.id,
+            type: 'window',
+            sandbox: undefined,
+          })
```

Comment:
nit: I'd prefer exposing `browsingContext.defaultRealm` here to hide the logic.

Reply:
This logic will eventually change since worker realms can be nested. I don't think this change matters too much.